src/v2/Components/LotTimer.tsx

```diff
@@ -0,0 +1,204 @@
+import { LotTimer_saleArtwork } from "v2/__generated__/LotTimer_saleArtwork.graphql"
+import { LotTimerQuery } from "v2/__generated__/LotTimerQuery.graphql"
+import { SystemContext } from "v2/System"
+import { SystemQueryRenderer } from "v2/System/Relay/SystemQueryRenderer"
+import { DateTime } from "luxon"
+import { Component, useContext } from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface Props {
+  saleArtwork: LotTimer_saleArtwork
+}
+
+export class LotTimer extends Component<Props> {
```

Comment:
We should always avoid class components in react these days

Reply:
Yes! I copied and pasted from `AuctionTimerLot` so thanks for the reminder to update the component. Made the following few lines way cleaner for ex.